### PR TITLE
fix compatibility with clang-format 18

### PR DESCRIPTION
**Pull request by Coeur**
```
✗ clang-format --version
clang-format version 18.1.1
```

### CODE REVIEW

**Comment by Coeur** [comment]
Hum, not compatible with clang-17. Let me try to add a comment to the line.

**Comment by GaryElshaw** [comment]
@ckerr Charles, is there a GitHub configuration option for non-maintainers to have merge rights for Transmission? I've mentioned this before, but the accruing backlog of pr's is often requiring an enormous amount of repeated re-work by contributors, authors and members.

I know your time is limited, and for that reason alone, i would like to nominate @tearfur having the ability to merge, with the requirement of maintainer approval. I won't speak for anyone else, but a personal observation i've noticed within myself, It's hard to move with further contributions if previous work sits in limbo.I'm often waiting for previous work to be merged to have the mental space to see where else my attention might be needed. 

Instead, there is a required remedial prioritisation. https://github.com/transmission/transmission/pull/6676#issuecomment-1991663517

**Review by ckerr — APPROVED** [review]

**Comment by ckerr** [comment]
Whatever the issue with sanitizer-tests is, it's affecting multiple PRs and is unrelated to the changes in 6690

**Pull request merged**
